src/todo.ts

```diff
@@ -0,0 +1,123 @@
+﻿/**
+* Interface that carries attributes about items in the list including;
+* Date Created
+* Description of Task
+* Date to due by
+* Unique ID for the task
+* Whether the task is done or not
+*/
+export interface ListItem {
+    createdAt: Date;
+    description: string;
+    dueAt: Date;
+    id: number;
+    isDone: boolean;
+}
+
+// class that is use to create todo lists
+export class Todo {
+    public list: ListItem[];
```

Comment:
I wanted list to be public so that instantiated objects would have access to the list. It could also be private with a function that returns it. Is there any difference? Always been confused about this.